e2e/ctl_v3_kv_test.go

```diff
@@ -192,6 +192,11 @@ func delTest(cx ctlCtx) {
 			[]string{"key", "--prefix"},
 			3,
 		},
+		{
+			[]kv{{"foo1", "bar"}, {"foo2", "bar2"}, {"foo3", "bar3"}},
+			[]string{"foo1", "--from-key"},
```

Comment:
actually i think this is still greater than foo1. probably we can try zoo1, zoo2, zoo3?